### PR TITLE
Qol 8053 GitHub actions

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

File: ckanext/datarequests/plugin.py [hunk 1]
```diff
@@ -17,41 +17,15 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with CKAN Data Requests Extension. If not, see <http://www.gnu.org/licenses/>.
 
-import ckan.lib.helpers as h
-import ckan.plugins as p
-import ckan.plugins.toolkit as tk
-import auth
```
Comment (duttonw) [inline ckanext/datarequests/plugin.py:23]: is the from ckanext.datarequests import better for reducing miss filename pickups?

File: ckanext/datarequests/plugin.py [hunk 2]
```diff
@@ -17,41 +17,15 @@
...
-import helpers
 import os
 import six
 import sys
 
-from functools import partial
-from pylons import config
-
-
-def get_config_bool_value(config_name, default_value=False):
```
Comment (duttonw) [inline ckanext/datarequests/plugin.py:35]: good to get these into a separate place

File: ckanext/datarequests/tests/test_plugin.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def setUp(self):
         self.tk_patch = patch('ckanext.datarequests.plugin.tk')
         self.tk_mock = self.tk_patch.start()
 
-        self.config_patch = patch('ckanext.datarequests.plugin.config')
```
Comment (duttonw) [inline ckanext/datarequests/tests/test_plugin.py:43]: does not seem like much of a change

File: ckanext/datarequests/validator.py [hunk 4]
```diff
@@ -17,12 +17,11 @@
...
 # along with CKAN Data Requests Extension. If not, see <http://www.gnu.org/licenses/>.
 
-import constants
-import ckan.plugins.toolkit as tk
-from ckanext.datarequests import db
-import plugin as datarequests
 import datetime
 
+import ckan.plugins.toolkit as tk
+from ckanext.datarequests import db, common, constants
```
Comment (duttonw) [inline ckanext/datarequests/validator.py:23]: much simpler with the common.

**Review by duttonw — APPROVED** [review]

File: ckanext/datarequests/plugin.py (hunk 1 above)
Comment (ThrawnCA) [reply ckanext/datarequests/plugin.py:23]: I'm not an expert on Python module loading, but it is more readable, anyway, making it immediately clear whether something is a sibling or a built-in module.

File: ckanext/datarequests/tests/test_plugin.py (hunk 3 above)
Comment (ThrawnCA) [reply ckanext/datarequests/tests/test_plugin.py:43]: The main reason for it was to break the circular dependency between `plugin`, `actions` and `validation`.

**Pull request merged**